examples/todoapp/common/compose-ui/build.gradle.kts

```diff
@@ -14,5 +16,11 @@ kotlin {
                 implementation(Deps.ArkIvanov.Decompose.extensionsCompose)
             }
         }
+
+        named("desktopMain") {
+            dependencies {
```

Comment:
Is it really needed with alpha3 and later?

Reply:
Seems like it works without the `ui-tooling` dependency. Should I remove it?

Reply:
Let's remove it, as we transitively depend on it [here](https://github.com/JetBrains/compose-jb/blob/696d20c931feb848f113de1e67fe9446adef538f/examples/todoapp/buildSrc/src/main/kotlin/multiplatform-compose-setup.gradle.kts#L31).

We have to [explicitly](https://github.com/JetBrains/compose-jb/blob/696d20c931feb848f113de1e67fe9446adef538f/templates/multiplatform-template/common/build.gradle.kts#L20) add it only if we don't depend on the aggregated dependency `compose.desktop.common`

Reply:
Done